### PR TITLE
Update docs theme

### LINKED ISSUE #764: [DOC] Update documentation website theme
### Context

The documentation currently uses very basic conda-store brand colors, but these need to be updated esp for dark mode.

### Value and/or benefit

Consistent and more polished website.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by pavithraes**
Fixes #764

## Description

This pull request updates the documentation theme properties with the conda-store theme.

Currently, this is a draft PR to have a preview as we make updates.

## Pull request checklist

- [ ] Did you test this change locally?
- [ ] Did you update the documentation (if required)?
- [ ] Did you add/update relevant tests for this change (if required)?

## Additional information

Add @smeragoel as a co-author while merging.

### CODE REVIEW

File: docusaurus-docs/src/css/custom.css [hunk 1]
```diff
@@ -6,34 +6,34 @@
...
-  --ifm-color-primary-light: #33925d;
-  --ifm-color-primary-lighter: #359962;
-  --ifm-color-primary-lightest: #3cad6e;
+  --ifm-color-primary: #298642;
+  --ifm-color-primary-dark: #206532;
+  --ifm-color-primary-darker: #144321;
+  --ifm-color-primary-darkest: #85CB97;
+  --ifm-color-primary-light: #5cb975;
+  --ifm-color-primary-lighter: #85cb97;
+  --ifm-color-primary-lightest: #d6eedc;
```
Comment (smeragoel) [inline docusaurus-docs/src/css/custom.css:15]:
```suggestion
  ---ifm-color-primary: #298642;
  ---ifm-color-primary-dark: #206532;
  ---ifm-color-primary-darker: #144321;
  ---ifm-color-primary-darkest: #0a2210;
  ---ifm-color-primary-light: #85cb97;
  ---ifm-color-primary-lighter: #addcba;
  ---ifm-color-primary-lightest: #d6eedc;
```

File: docusaurus-docs/src/css/custom.css [hunk 2]
```diff
@@ -6,34 +6,34 @@
...
-  --ifm-color-primary-light: #29d5b0;
-  --ifm-color-primary-lighter: #32d8b4;
-  --ifm-color-primary-lightest: #4fddbf;
+  --ifm-color-primary: #5cb975;
+  --ifm-color-primary-dark: #85cb97;
+  --ifm-color-primary-darker: #addcba;
+  --ifm-color-primary-darkest: #d6eedc;
+  --ifm-color-primary-light: #298642;
+  --ifm-color-primary-lighter: #206532;
+  --ifm-color-primary-lightest: #144321;
```
Comment (smeragoel) [inline docusaurus-docs/src/css/custom.css:28]:
```suggestion
--ifm-color-primary: #5cb975;
--ifm-color-primary-dark: #85cb97;
--ifm-color-primary-darker: #addcba;
--ifm-color-primary-darkest: #d6eedc;
--ifm-color-primary-light: #206532;
--ifm-color-primary-lighter: #144321;
--ifm-color-primary-lightest: #0a2210;
```

File: docusaurus-docs/src/css/custom.css [hunk 3]
```diff
@@ -6,34 +6,34 @@
...
-  --ifm-color-primary-light: #33925d;
-  --ifm-color-primary-lighter: #359962;
-  --ifm-color-primary-lightest: #3cad6e;
+  ---ifm-color-primary: #298642;
+  ---ifm-color-primary-dark: #206532;
+  ---ifm-color-primary-darker: #144321;
+  ---ifm-color-primary-darkest: #0a2210;
+  ---ifm-color-primary-light: #85cb97;
+  ---ifm-color-primary-lighter: #addcba;
+  ---ifm-color-primary-lightest: #d6eedc;
```
Comment (smeragoel) [inline docusaurus-docs/src/css/custom.css:15]:
```suggestion
   --ifm-color-primary: #298642;
   --ifm-color-primary-dark: #206532;
   --ifm-color-primary-darker: #144321;
   --ifm-color-primary-darkest: #0a2210;
   --ifm-color-primary-light: #85cb97;
   --ifm-color-primary-lighter: #addcba;
   --ifm-color-primary-lightest: #d6eedc;
```

**Comment by smeragoel** [comment]
I've made some color updates and currently, there are no accessibility issues detected, except for a minor one in the breadcrumbs. To address this, I recommend the following change:

From:
`--ifm-breadcrumb-color-active: var(--ifm-color-primary);`

To:
`--ifm-breadcrumb-color-active: var(--ifm-color-primary-dark);`

I'm not entirely sure which file needs this adjustment, so I'm hoping you could help out with this, @pavithraes.

Once this change is implemented, I plan to do a final check to ensure everything is in order, and then we should be all set!

**Comment by pavithraes** [comment]
@smeragoel - The changes you requested are done :)

**Comment by smeragoel** [comment]
@pavithraes the netlify preview isn't working for me, could you help out?

**Comment by pavithraes** [comment]
@smeragoel - Preview is available now :)

**Review by smeragoel — APPROVED** [review]
Looks great! There don't seem to be any major accessibility issues, however, I do think we could benefit from an a11y audit down the line.

**Pull request merged**